i18n/en.json

```diff
@@ -1002,7 +1002,7 @@
   },
   {
     "id": "api.emoji.upload.open.app_error",
-    "translation": ""
+    "translation": "Unable to upload the emoji. An error ocurred when tyring to open the attached image."
```

Comment:
Is this when you upload an image for the custom emoji?

Reply:
Yes, more precisely when you send the API request for create an emoji (which includes the image attached).